debezium-testing/debezium-testing-testcontainers/src/main/java/io/debezium/testing/testcontainers/ApicurioRegistryContainer.java

```diff
@@ -5,20 +5,74 @@
  */
 package io.debezium.testing.testcontainers;
 
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import javax.net.ssl.HttpsURLConnection;
+
 import org.testcontainers.containers.GenericContainer;
 import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
 
 public class ApicurioRegistryContainer extends GenericContainer<ApicurioRegistryContainer> {
 
-    private static final String DEBEZIUM_VERSION = DebeziumContainer.getDebeziumStableVersion();
+    private static final String APICURIO_VERSION = getApicurioVersionTestProperty();
     private static final Integer APICURIO_PORT = 8080;
+    private static final String TEST_PROPERTY_PREFIX = "debezium.test.";
 
     public ApicurioRegistryContainer() {
-        super("apicurio/apicurio-registry-mem:" + DEBEZIUM_VERSION);
+        super("apicurio/apicurio-registry-mem:" + (APICURIO_VERSION != null ? APICURIO_VERSION : getApicurioStableVersion()));
 
         this.waitStrategy = new LogMessageWaitStrategy()
                 .withRegEx(".*apicurio-registry-app.*started in.*");
 
         addExposedPort(APICURIO_PORT);
     }
+
+    public static String getApicurioVersionTestProperty() {
+        return System.getProperty(TEST_PROPERTY_PREFIX + "apicurio.version");
+    }
+
+    public static String getApicurioStableVersion() {
+        String APICURIO_VERSION = "";
+        try {
+            URL url = new URL("https://hub.docker.com/v2/repositories/apicurio/apicurio-registry-mem/tags/");
```

Comment:
Don't we have that code similarly elsewhere for Debezium version already? Would be nice to extract this into a (parameterized) method.

Reply:
yes @gunnarmorling, updated the changes.